pkg/provisioning/errors.go

```diff
@@ -17,5 +17,6 @@ package provisioning
 import "github.com/conduitio/conduit/pkg/foundation/cerrors"
 
 var (
-	ErrDuplicatedPipelineID = cerrors.New("duplicated pipeline ID")
+	ErrDuplicatedPipelineID    = cerrors.New("duplicated pipeline ID")
+	ErrDuplicatedAPIPipelineID = cerrors.New("duplicated pipeline ID already exists from API")
```

Comment:
I wouldn't name this "duplicated ID", it's more like we can't mutate the pipeline using a config file because it was created through the API. Essentially the counterpart to [this error](https://github.com/ConduitIO/conduit/blob/4be951a5374bbcc7212dd85c62706c0458679edd/pkg/orchestrator/errors.go#L24).